dace/sdfg/sdfg.py

```diff
@@ -460,6 +460,7 @@ class SDFG(OrderedDiGraph[SDFGState, InterstateEdge]):
                                  desc="Sub-array redistribution descriptors for this SDFG",
                                  to_json=_arrays_to_json,
                                  from_json=_arrays_from_json)
+    _with_scopes = Property(dtype=bool, default=False, desc='Contains state scopes')
```

Comment:
why is this a property? is it caching something?

Reply:
This was not supposed to be in this PR yet, I removed it again.